### PR TITLE
[Task #68249 #68253]Detail Salon

### PR DESCRIPTION
**Pull request by Hungnd-0370**
## Related Tickets:
- [#68249](https://edu-redmine.sun-asterisk.vn/issues/68249)
- [#68253](https://edu-redmine.sun-asterisk.vn/issues/68253)
## WHAT this PR does?
- Tạo BE FE cho chức năng xem thông tin chi tiết salon 

## Notes Impacted Areas:
- N/A.

## Attachments:
-
image

### CODE REVIEW

**Comment by Hungnd-0370** [comment]
ready

File: app/Http/Controllers/SalonController.php [hunk 1]
```diff
@@ -23,7 +24,7 @@ public function index()
             'salons/Salons',
             [
             [
-                'salons' => $this->calculateSalon(),
+                'salons' => $this->getAllSalon(),
             ],
```
Comment (ghost) [inline app/Http/Controllers/SalonController.php]: indent 4 spaces

File: app/Http/Controllers/SalonController.php [hunk 2]
```diff
@@ -105,7 +106,14 @@ function () use ($validated, $accepted_id, $active_id, $salon_active_id, $system
...
         }
 
-        return redirect()->route('salons.index');
+        return Inertia::render(
+            'salons/Salons',
+            [
+            [
+                'salons' => $this->calculateSalon(),
+            ],
+            ]
```
Comment (ghost) [inline app/Http/Controllers/SalonController.php]:
```suggestion
            [
                [
                    'salons' => $this->calculateSalon(),
                ],
            ]
```

File: resources/js/Pages/salons/Salons.jsx [hunk 3]
```diff
@@ -65,7 +65,9 @@ export default function Salons(props) {
                 return (
                     <div>
                         <div className='pb-2'>
+                            <a href={route('salons.show', {salon : record.id})}>
```
Comment (ghost) [inline resources/js/Pages/salons/Salons.jsx:68]: indent 4 spaces

File: resources/js/Pages/salons/Show.jsx [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+    const [salon, setSalon] = useState(props[0].salon);
+
+    const editSalon = (e) => {
+        
+    };
+
+    const showDeleteModal = (record) => {
+        const modalDelete = Modal.warning();
+        modalDelete.update({
+            title: "Delete Salon",
```
Comment (ghost) [inline resources/js/Pages/salons/Show.jsx]: localization

**Comment by Hungnd-0370** [comment]
ready

**Pull request merged**